google/cloud/aiplatform/metadata/artifact.py

```diff
@@ -32,7 +32,7 @@
 from google.cloud.aiplatform.metadata import resource
 from google.cloud.aiplatform.metadata import utils as metadata_utils
 from google.cloud.aiplatform.utils import rest_utils
-
+from google.cloud.aiplatform.metadata.types import base as types_base
```

Comment:
As discussed offline, let's rename "type" to "schema" to match our public docs.

Reply:
Done.